src/streaming/controllers/ScheduleController.js

```diff
@@ -379,6 +380,10 @@ function ScheduleController(config) {
         }
 
         setFragmentProcessState(false);
+        if (isNaN(e.index)) {
```

Comment:
Check for e.quality

Reply:
I've used `isNaN(e.index)` to check if it is an initialized segment. I don't get it but what should we do with `e.quality` ?